spring-xd-dirt/src/main/resources/META-INF/spring-xd/plugins/jmx/mbean-exporters.xml

```diff
@@ -22,8 +22,11 @@
 
 	<bean id="moduleObjectNamingStrategy"
 		class="org.springframework.xd.dirt.module.jmx.ModuleObjectNamingStrategy">
+		<constructor-arg ref="attributeSource" />
 		<constructor-arg name="domain" value="xd.${xd.stream.name:${xd.job.name:}}" />
 		<constructor-arg name="objectNameProperties" ref="objectNameProperties" />
 	</bean>
 
+	<bean id="attributeSource" class="org.springframework.jmx.export.annotation.AnnotationJmxAttributeSource" />
```

Comment:
In the other constructor you get a basic one with no bean factory; with this one we get a bean factory so we can resolve pph in the IMR.

I added this here so we get a value resolver in case (in future) some Mbean might have a PPH in an annotated object name but, I agree, I don't think it matters now - but I didn't think it would hurt to initialize the bean properly - in case something else changes in future.